kcidb/mq/__init__.py

```diff
@@ -15,8 +14,11 @@
 from google.api_core.exceptions import DeadlineExceeded
 import kcidb.io as io
 import kcidb.orm
-from kcidb import misc
+from kcidb.argparse import output_add_args
```

Comment:
Could we just do `import kcidb.argparse` and refer to `output_add_args` as `kcidb.argparse.output_add_args`?